ingestion/functions/parsing/argentina/argentina.py

```diff
@@ -0,0 +1,233 @@
+import os
+import sys
+from datetime import datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+
+def convert_date(date_str: str):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+
+    The date is listed in YYYYmmdd format, but the data server API will
+    assume that ambiguous cases (e.g. "05/06/2020") are in mm/dd/YYYY format.
+
+    Adding line to ensure date has type str
+    """
+    date = datetime.strptime(date_str, "%Y-%m-%d")
+
+    return date.strftime("%m/%d/%Y")
+
+
+def convert_gender(raw_gender):
+    if raw_gender == "M":
+        return "Male"
+    if raw_gender == "F":
+        return "Female"
+    return None
+
+
+def convert_location(entry):
+    '''
+    The only information we have is the province where case was diagnosed/hospitalised
+    '''
+
+    if entry['carga_provincia_nombre']:
+        return {"query": f"{entry['carga_provincia_nombre']}, Argentina"}
+    else:
+        return {"query": "Argentina"}
+
+
+def convert_age(entry):
+    '''
+    Want to return a float specifying age in years. If age field is empty, return None
+    '''
+    if entry['edad']:
+        if entry['edad_años_meses'] == 'Años':
+            return float(entry['edad'])
+        elif entry['edad_años_meses'] == 'Meses':
+            return float(float(entry['edad']) / 12)
+    else:
+        return None
+
+
+def get_confirmed_event(entry):
+    if entry['fecha_diagnostico']:
+        confirmation_date = convert_date(entry['fecha_diagnostico'])
+        note = 'Using Date of Diagnosis as the date of confirmation.'
+
+    elif entry['fecha_inicio_sintomas']:
+        confirmation_date = convert_date(entry['fecha_inicio_sintomas'])
+        note = 'Using Date of Symptom Onset as the date of confirmation, because Date of Diagnosis is missing.'
+
+    elif entry['fecha_apertura']:
+        confirmation_date = convert_date(entry['fecha_apertura'])
+        note = 'Using Date of Case Opening as the date of confirmation, because both Date of Diagnosis and Date of Symptom Onset are missing.'
+
+    if 'Caso confirmado por laboratorio' in entry['clasificacion']:
+        confirmed_value = 'Laboratory Test'
+    elif 'Caso confirmado por criterio clínico-epidemiológico' in entry['clasificacion']:
+        confirmed_value = 'Clinical Diagnosis'
+    else:
+        confirmed_value = 'Method Unknown'
+    return confirmation_date, confirmed_value, note
```

Comment:
return the event dictionary instead of those three fields so that it matches your method name